var/spack/repos/builtin/packages/py-pint/package.py

```diff
@@ -0,0 +1,20 @@
+# Copyright 2013-2018 Lawrence Livermore National Security, LLC and other
+# Spack Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+
+from spack import *
+
+
+class PyPint(PythonPackage):
+    """Pint is a Python package to define, operate and manipulate physical
+    quantities: the product of a numerical value and a unit of measurement.
+    It allows arithmetic operations between them and conversions from and
+    to different units."""
+
+    homepage = "https://pypi.python.org/pypi/pint"
+    url      = "https://pypi.io/packages/source/p/pint/Pint-0.8.1.tar.gz"
+
+    version('0.8.1', 'afcf31443a478c32bbac4b00337ee9026a13d0e2ac83d30c79151462513bb0d4')
+
+    depends_on('py-setuptools', type='build')
```

Comment:
@KineticTheory I was reading the README bundled with the [sources](https://github.com/pauleve/pint/archive/2018-07-10.zip):

> Pint requires the libRmath standalone library for the stochastic simulation and parameters
inference from time interval specifications.
You can disable this functionnality with the following command:
>
> 	$ python setup.py --disable-R

Are we missing something here (I see neither a dependency on R nor disabling of dependencies)?

Reply:
@alalazo This is for a different `pint` library. For this library, from the [README](https://github.com/hgrecco/pint/blob/master/README):

> - Dependency free: it depends only on Python and its standard library.

Reply:
Sorry, didn't check the link, just searched the name.